src/plugins/ovp/components/video-screen/DetailsTab.js

```diff
@@ -6,15 +6,34 @@ import AdvertisingFields from '@triniti/cms/plugins/common/components/advertisin
 import ImagePickerField from '@triniti/cms/plugins/dam/components/image-picker-field/index.js';
 import TaggableFields from '@triniti/cms/plugins/common/components/taggable-fields/index.js';
 import PicklistField from '@triniti/cms/plugins/sys/components/picklist-field/index.js';
+import useNode from '@triniti/cms/plugins/ncr/components/useNode.js';
 
 export default function DetailsTab(props) {
   const { nodeRef, node } = props;
   const schema = node.schema();
 
+  const { node: mezzanineNode } = useNode(node.get('mezzanine_ref'), false);
+  const mezzanineNodeStatus = mezzanineNode && mezzanineNode.has('transcoding_status') ? mezzanineNode.get('transcoding_status').getValue() : 'unknown';
```

Comment:
@rsumilang i'd make a Medialive component and only include it in detail if it has the mixin.  like how we do with other mixins.  we'd also only fetch the mezzanine node in that medialive component.

you'd have a single card component dedicated to the media live stuff, literally titled "Media Live".  reference the iam apps for example